src/platform/suecreek/include/platform/memory.h

```diff
@@ -398,7 +392,8 @@
 #define BOOT_LDR_BSS_SIZE		0x100
 
 /* Temporary stack place for boot_ldr */
-#define BOOT_LDR_STACK_BASE		HEAP_HP_BUFFER_BASE
+#define BOOT_LDR_STACK_BASE		(HP_SRAM_BASE + HP_SRAM_SIZE - \
```

Comment:
Why did it change from 0 to 0xBExxxxxx ?

Reply:
Because 0 is not realy a valid adress for a bootloader base.
Since HP heap is deleted and its base was used  by bootloader as a stack we had to come up with a different place to store it. Hence the calculation.